src/ContainerInstance/Az.ContainerInstance.psd1

```diff
@@ -95,7 +95,7 @@ CmdletsToExport = @()
 # VariablesToExport = @()
 
 # Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
-AliasesToExport = 'Get-AzContainerGroupOutboundNetworkDependencyEndpoint'
+AliasesToExport = 'Get-AzContainerGroupOutboundNetworkDependencyEndpoint', '*'
```

Comment:
Can '*' be removed?

Reply:
Sure. But I think it will not affect any thing. The reason why '*' is here is because template contains '*' in AliasesToExport.
https://github.com/Azure/azure-powershell/blob/d0d79fa9d5a645c83ee4418219a4e33c1338d873/tools/Gen2Master/Templates/Module.psd1#L81

When we sync gen to main branch, all aliases are appended to AliasesToExport. So '*' is kept in psd1.
https://github.com/Azure/azure-powershell/blob/d0d79fa9d5a645c83ee4418219a4e33c1338d873/tools/Gen2Master/MoveFromGeneration2Master.ps1#L321